docs: update Seafile OIDC integration (#4151)
authelia/authelia: docs/content/en/integration/openid-connect/seafile/index.md

* __Client ID:__ `seafile`
* __Client Secret:__ `seafile_client_secret`

> *__Important Note:__ [Seafile] only uses email to identify unique user accounts[^1]. When using the
> [File First Factor](../../../configuration/first-factor/file.md) there may be multiple Authelia users with the same
> email, resulting in them being assigned the same [Seafile] account. This issue could be mitigated by tuning
> the `OAUTH_ATTRIBUTE_MAP` or modifying `seahub/seahub/oauth/views.py` in your [Seafile] installation[^2].*
> 

Reviewer: ```suggestion
```

## Configuration

### Application

To configure [Seafile] to utilize Authelia as an [OpenID Connect] Provider:

> 1. Install the `requests_oauthlib` pip package which is required but not installed with [Seafile] by default[^3].

Reviewer: ```suggestion
1. Install `requests_oauthlib` which is required but not installed with [Seafile] by default. See the [Seafile] documentation in the [see also](#see-also) section for more information.
```

OAUTH_TOKEN_URL = 'https://auth.example.com/api/oidc/token'
OAUTH_USER_INFO_URL = 'https://auth.example.com/api/oidc/userinfo'
OAUTH_SCOPE = [
    "openid",
    "profile",
    "email",
]
OAUTH_ATTRIBUTE_MAP = {
    "email": (True, "email"),
    "name": (False, "name"),

>     "id": (False, "not used"),

Reviewer: ```suggestion
```

}
```

> Remember to restart [Seafile] so your configuration changes take effect.
> 

Reviewer: ```suggestion
```

* [Seafile OAuth Authentication Documentation](https://manual.seafile.com/deploy/oauth/)

> [^1]:https://forum.seafile.com/t/oauth-question-error-with-sso/5481/2
> [^2]:https://forum.seafile.com/t/oauth-question-error-with-sso/5481/3
> [^3]:https://manual.seafile.com/deploy/oauth/#oauth
> 

Reviewer: ```suggestion
```
